plugins/woocommerce/tests/e2e-pw/tests/shopper/launch-your-store.spec.js

```diff
@@ -3,8 +3,21 @@ const { setOption } = require( '../../utils/options' );
 
 test.describe(
 	'Launch Your Store front end - logged out',
-	{ tag: [ '@payments', '@services' ] },
+	{ tag: [ '@payments', '@services', '@local' ] },
 	() => {
+		test.afterEach( async ( { baseURL } ) => {
+			try {
+				await setOption(
+					request,
+					baseURL,
+					'woocommerce_coming_soon',
+					'no'
+				);
+			} catch ( error ) {
+				console.log( error );
+			}
+		} );
```

Comment:
Why is this needed? And why not letting it fail is the request fails?

Reply:
This is required in order to test against external envs. It fails to switch Coming Soon mode properly without this. Can't remember all the details, but without it, it will be false-positive.

And probably I copy/pasted existing code with `try` to be in `afterEach`. I can exclude it.

Reply:
You mean any following tests that need that feature turned off will fail. 
I think we should use fixtures or before hooks to properly set up tests instead of relying on other test properly cleaning up. If this happens, this afterEach is not required.

Reply:
The second test when switching from LYS to LYS with stores only, it should be first turned OFF and then turned ON with stores only. This goes to API change and live site, not a real issue when manually switching modes.

I changed from `afterEach` to `beforeEach`.